Source/ModelGroup.cs

```diff
@@ -115,8 +115,19 @@ public static List<int> GetSinglePlaneIndices()
                 };
             }
 
-            public static List<Runtime.MeshPrimitive> CreateMultiPrimitivePlane()
+            public static List<Runtime.MeshPrimitive> CreateMultiPrimitivePlane(bool includeTextureCoords = true)
             {
+                List<List<List<Vector2>>> textureCoordSets = new List<List<List<Vector2>>>()
```

Comment:
This still seems weird to me. How about split GetMultiPrimitivePlaneTextureCoordSets into two and set the TextureCoordSets only when includeTextureCoords is true?

Reply:
It is already only being set when includeTextureCoords is true. You mean to not do the set null bit?

Reply:
I mean why create an array when you are only going to use it as separated entries.